docs/user_guide/datasets.rst

```diff
@@ -0,0 +1,333 @@
+and for a time was included as one of `scikit-learn's <https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_boston.html>`
+ and tensorflow's standard toy datasets. It has also been the benchmark of 
+choice for `many <https://ieeexplore.ieee.org/abstract/document/8556738/>` 
+`machine <https://lib.dr.iastate.edu/cgi/viewcontent.cgi?article=1187&context=imse_conf>` 
+`learning <https://proceedings.neurips.cc/paper/1999/file/f3144cefe89a60d6a1afaf7859c5076b-Paper.pdf>`
+`papers <https://arxiv.org/search/?query=boston+housing&searchtype=all&source=header>`.
+As of scikit-learn version 1.2, the dataset has been removed.
+
+The dataset contains the following columns:
+
+============ ==================
+Column Name   Description                                                          
+============ ==========================================================================
+CRIM         per capita crime rate by town                                         
+ZN           proportion of residential land zoned for lots over 25,000 sq.ft.
+INDUS        proportion of non-retail business acres per town
+CHAS         Charles River dummy variable (= 1 if tract bounds river; 0 otherwise)
+NOX          nitric oxides concentration (parts per 10 million)
+RM           average number of rooms per dwelling
+AGE          proportion of owner-occupied units built prior to 1940
+DIS          weighted distances to five Boston employment centers
+RAD          index of accessibility to radial highways
+TAX          full-value property-tax rate per $10,000
+PTRATIO      pupil-teacher ratio by town
+B            1000(Bk - 0.63)^2 where Bk is the proportion of blacks by town
+LSTAT        % lower status of the population
+MEDV         Median value of owner-occupied homes in $1000’s
+============ ============================================================================
+
+The cells below show basic summary statistics about the data, the dtypes of the 
+columns, and the number of missing values. Note that the 
+:func:`fairlearn.datasets.fetch_boston` function warns user by default that the
+dataset contains fairness issues. Setting :code:`warn=False` will turn the
+warning off. To return the dataset as a data frame, pass 
+:code:`as_frame=True` and call the *data* attribute.
+
+For more information about how to use the :code:`fetch_boston` function, 
+visit the `fairlearn datasets documentation 
+<https://fairlearn.org/v0.7.0/api_reference/fairlearn.datasets.html#id15>`. 
+
+.. note::
+
+    Calling the :func:`fairlearn.datasets.fetch_boston` function will raise a 
+    :code:`FairnessWarning`.
+    For more information on this warning refer to
+    `https://fairlearn.org/v0.7.0/api_reference/fairlearn.datasets.html 
+    <https://fairlearn.org/v0.7.0/api_reference/fairlearn.datasets.html>`_.
+
+.. doctest:: datasets
+
+    >>> from fairlearn.datasets import fetch_boston
+    >>> import pandas as pd
+
```

Comment:
Does this work properly with blank lines? Maybe try running it and paste a screenshot if it works. I'd be very surprised if it does.

Reply:
I removed the blank lines so hopefully it will work!

Reply:
UPDATE: removing the blank lines does render the code correctly. Here's a screenshot! I will resolve this comment. 
![image](https://user-images.githubusercontent.com/15962391/136202570-ef75ca0e-44e8-4951-a931-4a4b680f7a1e.png)